server/src/db.py

```diff
@@ -1,40 +1,71 @@
-import os
+import logging
 
-from motor import motor_asyncio
+import asyncpg
+from aiohttp import web
+from asyncpg.pool import Pool
 
+from config import BASE_DIR
 
-DB_CONNECTION = None
-DATABASE = None
 
+log = logging.getLogger(__name__)
 
-def db_connection():
-    global DB_CONNECTION
-    if DB_CONNECTION is None:
-        DB_CONNECTION = motor_asyncio.AsyncIOMotorClient(os.environ['MONGO_URL'])
-    return DB_CONNECTION
 
+def get_query(query_name: str) -> str:
+    try:
+        with open(BASE_DIR / f'queries/{query_name}.sql', 'r') as file:
+            return file.read()
+    except FileNotFoundError:
+        log.error(f'Query {query_name} not found')
+        return ''
 
-def db() -> motor_asyncio.AsyncIOMotorDatabase:
-    global DATABASE
-    if DATABASE is None:
-        DATABASE = db_connection()[os.environ['MONGO_DATABASE']]
-    return DATABASE
 
+async def create_db(app: web.Application):
+    log.info('create_db')
+    pool: Pool = app['pool']
+    async with pool.acquire() as con:
+        await con.execute(get_query('init'))
 
-async def create_room(data: dict):
-    await db().rooms.insert_one(data)
 
+async def create_room(pool: Pool, data: dict) -> int:
+    async with pool.acquire() as con:
+        room_id = await con.fetchval(
```

Comment:
Here you make two queryes, but you can make single one, create with massword, just use get method of dick, to not fall, if there is no password, or make validation, that is always password in data